test/fixtures/poro.rb

```diff
@@ -258,4 +260,34 @@ def maker
   cache only: [:id]
   attributes :id
 end
+
+class TweetSerializer < ActiveModel::Serializer
+  attributes :id, :body, :date
+
+  belongs_to :author
+  class AuthorSerializer < ActiveModel::Serializer
+    attributes :id
+  end
```

Comment:
How is this different from `belongs_to: :author, serializer: TweetAuthorSerializer`?